atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/metrics/SweepMetricsAssert.java

```diff
@@ -103,6 +109,26 @@ public void hasMillisSinceLastSweptThoroughEqualTo(long value) {
         objects.assertEqual(info, getGaugeThorough(AtlasDbMetricNames.LAG_MILLIS).getValue(), value);
     }
 
+    public void hasEntriesReadInBatchConservativeEqualTo(long... outcomes) {
+        arrays.assertContainsExactlyInAnyOrder(info,
+                getBatchSizeHistogram(SweepStrategy.CONSERVATIVE).getSnapshot().getValues(),
+                outcomes);
+    }
```

Comment:
nit: I'd probably name this just `hasEntriesReadInBatchConservative(...)` or even `containsEntriesReadInBatchConservative(...)` for similarity with the existing assertions. Might just be me, but `EqualTo` makes me expect a single following value (or that the thing is equal to all of the vararg parameters)

Reply:
Good call